src/client/renderingengine.h

```diff
@@ -46,6 +46,10 @@ class RenderingCore;
 class RenderingEngine
 {
 public:
+	/// Default color factor before applying effects like bloom or tomemapping
+	/// this is derived from tonemapping code and tuned empirically
```

Comment:
Here, too.

Reply:
This is standard syntax for doxygen comments, can be used interchangeably with `/**`